Simplified/en.lproj/Localizable.strings

```diff
@@ -148,6 +148,10 @@
 "Next Chapter" = "Next Chapter";
 
 // NYPLReaderSettingsView
+"SpecialFontDescriptionTitle" = "Special Font";
```

Comment:
For all the new strings, since we don't have localizations for Spanish and German, `SpecialFontDescriptionTitle` will be used instead of the actual localization. So it's better to use the actual English localization as key, as a fallback, so:
`"Special Font" = "Special Font";`

Reply:
Italian localizations:

"Special Font" = "Font speciale" (btw where is this used in the UI?)
"Sans" = "Sans";
"Serif" = "Serif";
"Open dyslexic" = "Font per dislessia";